### DISCUSSION TITLE
Tree style lists

### DISCUSSION THREAD

**Discussion by ryanbr** [uBlockOrigin/uBlock-issues#2686] [General] [answered]
Not sure if sublist is a good change, if a new user wants Fanboy Annoyances. Do they have to guess which sublists are used by it?

Maybe a tree style with drop down would be a better solution, if a user wants more granual control.

```

[ ] Adguard Annoyances
  |_
  |_
  |_
  ...

[ ] Fannboy Annoyances
  |_
  |_
  |_
  ...

```

**Comment by uBlock-user** [comment, marked as answer]
Already implemented in the dev build.

![Capture](https://github.com/uBlockOrigin/uBlock-issues/assets/21290713/6cefe93c-d0be-4d76-a155-a23e814a34a3)

**Reply by ryanbr** [reply]
Surprised this didn't roll out initially.
